style.css

```diff
@@ -509,6 +542,54 @@ textarea {
   text-align: center;
 }
 
+.popup-container {
+  width: 100vw;
+  height: 100vh;
+  position: fixed;
+  backdrop-filter: brightness(0.5);
+}
```

Comment:
- [ ] Please 🙏🏽 use `overflow-y: scroll` to be able to see all the modal windows for a better UX.